### DISCUSSION TITLE
[A-L] (2023/24) Foglio 7 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#183] [Esercizi - Bertini]
![Screenshot 2023-11-13 194655](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/b72ae7c4-8acd-4b36-a301-49111f91ea6b)

**Comment by Elia-Belli** [comment]
Consideriamo lo schema di Bernoulli codificato con $0,1$ che indicano rispettivamente croce e testa
- **(1)** Calcoliamo i casi inziali presenti nel testo:
$\mathbb{P}(X=1)=\mathbb{P}(10)+\mathbb{P}(01)=p\cdot (1-p)+(1-p)\cdot p$
$\mathbb{P}(X=2)=\mathbb{P}(110)+\mathbb{P}(001)=p^2\cdot (1-p)+(1-p)^2\cdot p$
Siano $T,C$ v.a. che indicano rispettivamente dopo quanti lanci abbiamo ottenuto una testa/croce, osserviamo che $T\sim Geom(p)$ e $C\sim Geom(1-p)$ allora:
$$\mathbb{P}(X=k)=\mathbb{P}(T=k+1)+\mathbb{P}(C=k+1)=p^k(1-p)+(1-p)^kp$$

- **(2)** Calcolo diretto per il valore atteso (divise le somme il calcolo è analogo a quello del valore attesso per una v.a. geometrica):
$$\mathbb{E}(X)=\sum^\infty\_{k=1}k\cdot \mathbb{P}(X=k)=\sum^\infty_{k=1}k\cdot [p^k\cdot (1-p)+(1-p)^k\cdot p]=(1-p)\cdot \sum^\infty_{k=1}k\cdot p^k+p\cdot \sum^\infty_{k=1}k\cdot (1-p)^k=$$

$$=\frac{(1-p)}{p}+\frac{p}{(1-p)}=\frac{(1-p)^2+p^2}{p(1-p)}=\frac{2p^2-2p+1}{p(1-p)}$$

**Comment by LeoCosta02** [comment]
@Elia-Belli  E[X] non dovrebbe essere solo E [T] +E[C]?
Ho calcolato E[X] per X=1 e X=2 e come risultato combacia se calcolo E [T] +E[C] per T,C=1,2 mentre se divido per due E [T] +E[C] il risultato é la metà di  E[X]

**Comment by Elia-Belli** [comment]
@LeoCosta02 non puoi calcolare $\mathbb{E}(X)$ per dei valori di $X$ fissati, il valore di attesa è una "media pesata"

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/5c3ee498-867e-4011-8461-b673346a7f02)

**Comment by LeoCosta02** [comment]
@CiottoloMaggico , @Elia-Belli  come avete fatto a trasformare le due sommatorie ?

**Comment by CiottoloMaggico** [comment]
https://it.m.wikipedia.org/wiki/Lista_delle_serie_matematiche
Sono tutte qui tra le serie di potenze con ABS(x) < 1

**Comment by LeoCosta02** [comment]
@CiottoloMaggico grazie
